pkg/virt-api/rest/kubeproxy.go

```diff
@@ -48,14 +48,15 @@ type ResponseHandlerFunc func(rest.Result) (interface{}, error)
 
 func GroupVersionProxyBase(ctx context.Context, gv schema.GroupVersion) (*restful.WebService, error) {
 	ws := new(restful.WebService)
+	ws.Doc("The KubeVirt API, a virtual machine management.")
 	ws.Path(GroupVersionBasePath(gv))
 
 	virtClient, err := kubecli.GetKubevirtClient()
 	if err != nil {
 		return nil, err
 	}
 	autodiscover := endpoints.NewHandlerBuilder().Get().Decoder(endpoints.NoopDecoder).Endpoint(NewAutodiscoveryEndpoint(virtClient.RestClient())).Build(ctx)
-	ws.Route(ws.GET("/").Produces(mime.MIME_JSON).Writes(metav1.APIResourceList{}).To(endpoints.MakeGoRestfulWrapper(autodiscover)))
+	ws.Route(ws.GET("/").Produces(mime.MIME_JSON).Writes(metav1.APIResourceList{}).To(endpoints.MakeGoRestfulWrapper(autodiscover)).Doc("Get KubeVirt API Resources"))
```

Comment:
I would like to set `Operation` attribute. do you have any preference?
`list_resources` / `get_resources`